static/js/controllers/help.js

```diff
@@ -1,28 +1,30 @@
-(function () {
+'use strict';
 
-  'use strict';
+var inboxControllers = angular.module('inboxControllers');
 
-  var inboxControllers = angular.module('inboxControllers');
+inboxControllers.controller('HelpCtrl',
+  ['$scope', '$stateParams', '$q', 'DB', 'Markdown', 'UserSettings',
+  function ($scope, $stateParams, $q, DB, Markdown, UserSettings) {
+    $scope.filterModel.type = 'help';
+    $scope.loading = true;
 
-  inboxControllers.controller('HelpCtrl',
-    ['$scope', 'Session', 'Debug', 'DB',
-    function ($scope, Session, Debug, DB) {
-      $scope.filterModel.type = 'help';
-      $scope.url = window.location.hostname;
-      $scope.userCtx = Session.userCtx();
-      $scope.reload = function() {
-        window.location.reload(false);
-      };
-      $scope.enableDebugModel = {
-        val: Debug.get()
-      };
-      $scope.$watch('enableDebugModel.val', Debug.set);
-      DB.get().info().then(function (result) {
-        $scope.dbInfo = JSON.stringify(result, null, 2);
-      }).catch(function (err) {
-        console.error('Failed to fetch DB info', err);
-      });
-    }
-  ]);
+    var docId = 'help:' + $stateParams.page;
 
-}());
+    DB.get().get(docId)
+      .then(function(doc) {
+        new $q(function(resolve) {
+          UserSettings(function(err, settings) {
```

Comment:
Use the Language service here - it gets the user's chosen language in a standardised way.